articles/email/liquid-syntax.md

```diff
@@ -24,12 +24,46 @@ There are two types of markup in Liquid: **output** and **tag**.
 
 `Hello {{ name }}!`
 
-You can further customize the appearance of the output by using [filters](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters), which are simple methods.
+You can further customize the appearance of the output by using filters, which are simple methods. For example, the `upcase` filter will convert the text which is passed to the filter to upper case:
 
-`Hello {{name | upcase}}!`
+`Hello {{ name | upcase }}!`
 
 Multiple filters are separated by `|` and are processed from left to right, applying the subsequent filter to the result of the previous one. The template will render the final result.
 
+The following filters are supported:
```

Comment:
@jerriep Maybe replace the bullet list with a matrix? I think it'll look more clean.

| Filter | Description | Example |
|------|-------------|----------|
| `append` | append a string | `{{ 'foo' | append:'bar' }} #=> 'foobar'` |
| `capitalize` | capitalize words in the input sentence | `{{ "my great title" | capitalize }} #=> My great title` |